src/model/Bet.tsx

```diff
@@ -3,5 +3,6 @@ export interface Bet {
     userUuid: string | undefined;
     gameId: number;
     gameResult: string;
+    betType: string;
```

Comment:
Change it to type (just because I like it that way).

```suggestion
    type: string;
```